conf/postgres-operator/pgadmin-service-template.json

```diff
@@ -0,0 +1,26 @@
+{
+        "kind": "Service",
+        "apiVersion": "v1",
+        "metadata": {
+            "name": "{{.Name}}",
+            "labels": {
+                "vendor": "crunchydata",
+                "name": "{{.Name}}",
+                "pgadmin": "true",
+                "pg-cluster": "{{.ClusterName}}"
+            }
+        },
+        "spec": {
+            "ports": [{
+                "protocol": "TCP",
+                "port": {{.Port}},
+                "targetPort": {{.Port}},
+                "nodePort": 0
+            }],
+            "selector": {
+                "name": "{{.Name}}"
+            },
+            "type": "ClusterIP",
```

Comment:
Just a note that in the long run it may be beneficial to allow the service type to be changed/configurable, e.g. to use `NodePort`, `LoadBalancer`, etc.

Reply:
To consider for the future :wink: